lang/cs/Org.Apache.REEF.Client/YARN/LegacyJobResourceUploader.cs

```diff
@@ -40,26 +40,26 @@ internal sealed class LegacyJobResourceUploader : IJobResourceUploader
             @"org.apache.reef.bridge.client.JobResourceUploader";
 
         private readonly IJavaClientLauncher _javaLauncher;
-        private readonly IJobSubmissionDirectoryProvider _jobSubmissionDirectoryProvider;
         private readonly IResourceArchiveFileGenerator _resourceArchiveFileGenerator;
         private readonly IFile _file;
 
         [Inject]
-        private LegacyJobResourceUploader(IJavaClientLauncher javaLauncher,
-            IJobSubmissionDirectoryProvider jobSubmissionDirectoryProvider,
+        private LegacyJobResourceUploader(
```

Comment:
Yes, this preserves the exact behavior as with current Java based submission for resource uploading.